config/tree_nodes_model.xml

```diff
@@ -19,7 +19,9 @@
         </Action>
         <Action ID="SetupMTCWaveHand">
             <!-- The MoveIt Studio UI will use this tag to organize Behaviors in the sidebar. -->
-            <metadata subcategory="MTC"/>
+            <MetadataFields>
+                <Metadata subcategory="MTC"/>
```

Comment:
We made this switch in the core repo a while back, so while not related to your changes, we should do it.

```suggestion
                <Metadata subcategory="Task Planning"/>
```